pkgs/development/libraries/qt-jdenticon/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchFromGitHub, mkDerivation, qmake, qtbase }:
+
+mkDerivation rec {
+  pname = "qt-jdenticon";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Nheko-Reborn";
+    repo = "qt-jdenticon";
+    rev = "v${version}";
+    sha256 = "1dk2cmz34pdn79k6lq6jlbvy0gfp0w2vv1w4f8c9xmgmf6brcxq1";
+  };
+
+  nativeBuildInputs = [ qmake ];
+  buildInputs = [ qtbase ];
+
+  dontWrapQtApps = true;
```

Comment:
Why do you need this?

Reply:
I quote a paragraph from `doc/languages-frameworks/qt.section.md` file:

``` html
    <para>
      All Qt packages must include <literal>wrapQtAppsHook</literal> in
      <literal>nativeBuildInputs</literal>, or you must explicitly set
      <literal>dontWrapQtApps</literal>.
    </para>
```

Before I added this option the build of the package was producing an empty directory. If I’m not mistaken just by adding this option it made everything work as expected.

Reply:
> I quote a paragraph from doc/languages-frameworks/qt.section.md file:

Fair enough, I forgot :).